tests/core/policies/test_rule_policy.py

```diff
@@ -168,7 +300,10 @@ def test_incomplete_rules_due_to_slots():
         policy.train([complete_rule, incomplete_rule], domain, RegexInterpreter())
     assert all(
         name in execinfo.value.message
-        for name in {some_action, incomplete_rule.sender_id,}
+        for name in {
+            some_action,
+            incomplete_rule.sender_id,
+        }
```

Comment:
what's wrong with keeping it on one line?
```suggestion
        for name in {some_action, incomplete_rule.sender_id}
```

Reply:
Ask the formatter tool I'd say :-D But it's true that I should've checked the changes made by `black` to the code. @wochinge any thoughts?

Reply:
For now, I've pushed without fixing this. Partly because I didn't want to mark the relevant lines as `# fmt:off/on` (because to me this smells as it would need to be done everywhere), and because I didn't want to push something that doesn't pass the `black` checks.

Reply:
what do you mean? my suggestion should pass the `black` check

Reply:
also, just do `black .` in your rasa repo, it'll do the job automatically for you

Reply:
except removing a comma (but this you can do manually if you spot it)

Reply:
@Ghostvv the changes were made by my pre-commit hook, not by me. I've just checked and it looks like my pre-commit hook uses maybe a different version (or config) of `black` because the pre-commit prefers the multi-line form of the set expression above while `make formatter` (i.e. `black rasa tests`) prefers the single-line version. I have no idea what to do...

Reply:
did you try updating `black`? I have no idea what is "pre-commit hook" 😅

Reply:
I've solved it by deleting the pre-commit hook altogether because it was actually using a newer version of `black` than `make formatter`. Those two versions will have to be synced though because in the repo readme [here](https://github.com/RasaHQ/rasa#formatting) we tell people how to use `black` in a pre-commit hook (so that reformatting is done automatically when you `git commit` and you don't have to run it manually).